src/views/GovernanceOverview/index.scss

```diff
@@ -0,0 +1,41 @@
+.easi-governance-overview {
+  margin-bottom: 36px;
+
+  &__text {
+    font-size: 22px;
+    line-height: 36px;
+  }
+
+  &__indented-wrapper {
+    border-left: 4px solid #585C62;
+    padding-top: 6px;
+    padding-bottom: 6px;
+    margin-block-start: 1em;
```

Comment:
I've never understood `margin-block-start` rule. Can it just be `margin: 1em 0;`